llvm/include/llvm/CodeGen/SelectionDAGNodes.h

```diff
@@ -1080,6 +1080,7 @@ END_TWO_BYTE_PACK()
   void printrWithDepth(raw_ostream &O, const SelectionDAG *G = nullptr,
                        unsigned depth = 100) const;
 
+#if !defined(NDEBUG) || defined(LLVM_ENABLE_DUMP)
```

Comment:
I also found SystemZAddressingMode was calling one these functions outside of a debug build, but the only use was indirectly in a LLVM_DEBUG macro.